fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -195,6 +197,31 @@
         tripleFormat.setTimeZone(getTimeZone("GMT"));
     }
 
+    private static File TEST_FILE;
```

Comment:
If that is the case than we need to actually write a mechanism to ensure that (using a streaming request) instead of gambling on a fixed size. A magic chosen fixed size will eventually fail, perhaps months or years from now, and it will be almost impossible to understand why.